internals/plan/plan.go

```diff
@@ -917,6 +1031,15 @@ func (p *Plan) Validate() error {
 	if err != nil {
 		return err
 	}
+
+	// Each section extension must validate the combined plan.
+	for field, extension := range layerExtensions {
+		err = extension.ValidatePlan(p)
+		if err != nil {
+			return fmt.Errorf("cannot validate plan section %q: %w", field, err)
```

Comment:
Same here.

Reply:
Removed wrapping of the error. Thank you. Resolved.